taskTester.ts

```diff
@@ -0,0 +1,14 @@
+const { ToadScheduler, SimpleIntervalJob, Task } = require('toad-scheduler')
```

Comment:
I think this file is no longer needed

Reply:
@kibertoad well I tried to add to ci.yml but that didn't work... it fails all the CI tests now.